examples/rbfe_edge_list.py

```diff
@@ -66,8 +67,12 @@ def read_from_args():
             print(f"Submitting job for {mol_a_name} -> {mol_b_name}")
             mcs_threshold = 2.0
             core, smarts = atom_mapping.get_core_with_alignment(mol_a, mol_b, threshold=mcs_threshold)
-            fut = cpc.submit(run_pair, mol_a, mol_b, core, forcefield, protein, args.n_frames, args.seed + row_idx)
-            futures.append(fut)
+            cfutures.append(
+                cpc.submit(run_complex, mol_a, mol_b, core, forcefield, protein, args.n_frames, args.seed + row_idx)
+            )
+            sfutures.append(
```

Comment:
does it make sense to append sfutures first? (since they're faster and will likely be finished first)